src/app/components/Billboard/index.styles.ts

```diff
@@ -0,0 +1,100 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '../../utilities/pixelsToRem';
+
+export default {
+  link: ({ palette }: Theme) =>
+    css({
+      textDecoration: 'none',
+      '&:hover, &:focus': {
+        color: palette.WHITE,
+        textDecoration: 'underline',
+      },
+    }),
+  headerContainer: ({ mq }: Theme) =>
+    css({
+      display: 'flex',
+      flexDirection: 'column',
+      position: 'relative',
+      [mq.HIGH_CONTRAST]: {
+        borderBottom: `solid ${pixelsToRem(1)}rem transparent`,
+      },
+    }),
+  backgroundContainer: ({ palette }: Theme) =>
+    css({
+      position: 'absolute',
+      top: '0',
+      backgroundColor: palette.GREY_10,
+      bottom: '0',
+      width: '100%',
+      overflow: 'hidden',
+    }),
+  contentContainer: ({ mq, gridWidths }: Theme) =>
+    css({
+      [mq.GROUP_4_MIN_WIDTH]: {
+        maxWidth: `${pixelsToRem(gridWidths[1280])}rem`,
+        margin: '0 auto',
+        position: 'relative',
+        width: '100%',
+      },
+      '::before': {
+        position: 'absolute',
+        // bottom: 0,
+        // top: 0,
+        // left: 0,
+        // right: 0,
```

Comment:
Can be removed?

Reply:
We could do - we copied this from the promos on the homepage to try and make the whole component area clickable, but we can't quite get it working so we're going to focus on this as part of the next ticket: https://jira.dev.bbc.co.uk/browse/WSTEAMA-1081. So, it will get deleted or moved then.